test/test_proxy_tensor.py

```diff
@@ -814,7 +814,7 @@ def fused_adam(params, grads, exp_avgs, exp_avg_sqs, max_exp_avg_sqs, state_step
                 exp_avg_sqs,
                 max_exp_avg_sqs,
                 state_steps,
-                lr=0.1,
+                lr=torch.tensor(0.1),
```

Comment:
Both should work no?

Reply:
This calls the private API directly so it does not pass the schema check. i.e., it fails to compile and would not work.